### PR TITLE
Resuscitated Arbitrum Sepolia to Sepolia devnet deployment

### PR DESCRIPTION
**Pull request by mani99brar**
- Updated AMB addresses for deployments.
- Redeployment of Arbitrum Sepolia to Sepolia route contracts (devnet).
- New deployments of subgraphs for inbox and outbox of arbSep->Sep devnet (Veascan).
- New deployment of subgraph for "relayer-cli" (devnet).
- Updated lightbulb switch address for devnet relay example.

---

## PR-Codex overview
This PR focuses on updating various configurations, addresses, and dependencies across multiple files, enhancing the functionality and compatibility of the project. It also introduces new environment variables for subgraph endpoints and makes changes to the contract deployments.

### Detailed summary
- Added a new JSON file `11155111.json` with `ts` and `nonce`.
- Updated `dotenv` version in `package.json` files.
- Changed `amb` addresses in multiple contract deployment files.
- Updated `start` and `build` scripts in `veascan-web/package.json`.
- Introduced new environment variables for subgraph endpoints.
- Modified `getInboxSubgraph` function to use environment variables.
- Updated `CODEOWNERS` with new contributors.
- Changed deployment addresses in `VeaInboxArbToEthDevnet.json`.

> The following files were skipped due to too many changes: `contracts/deployments/arbitrumSepolia/VeaInboxArbToEthDevnet.json`, `yarn.lock`, `contracts/deployments/sepolia/VeaOutboxArbToEthDevnet.json`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

## Summary by CodeRabbit

- **New Features**
	- Enhanced management of different environments (devnet, testnet, mainnet) with specific scripts for starting, building, and generating code tailored to each environment.
	- Added support for custom arguments in ShellCheck configuration.
- **Bug Fixes**
	- Upgraded various dependencies, which may include bug fixes and performance improvements.
- **Documentation**
	- Updated `.gitignore` to prevent tracking of new environment configuration files.
- **Chores**
	- Cleaned up and organized scripts for better usability across different deployment environments.
	- Updated ownership assignments in GitHub configuration files.

### CODE REVIEW

File: contracts/deploy/01-outbox/01-arb-sepolia-to-chiado-outbox.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ const paramsByChainId = {
     epochPeriod: 1800, // 30 min
     minChallengePeriod: 0, // 30 min
     numEpochTimeout: 10000000000000, // never
-    amb: "0x99Ca51a3534785ED619f46A79C7Ad65Fa8d85e7a",
+    amb: "0x8448E15d0e706C0298dECA99F0b4744030e59d7d", // https://docs.gnosischain.com/bridges/About%20Token%20Bridges/amb-bridge#key-contracts
```
Comment (fcanela) [inline contracts/deploy/01-outbox/01-arb-sepolia-to-chiado-outbox.ts:18]: ignoreable Nice comment, it made it very easy to figure out where the value came from

File: relayer-cli/src/utils/proof.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ const getMessageDataToRelay = async (chainid: number, nonce: number) => {
     const subgraph = getSubgraph(chainid);
 
     const result = await request(
-      `https://api.studio.thegraph.com/query/67213/${subgraph}/version/latest`,
+      `https://api.studio.thegraph.com/query/85918/${subgraph}/version/latest`,
```
Comment (fcanela) [inline relayer-cli/src/utils/proof.ts:8]: Consider extracting this into an environment variable, if not the whole prefix at least the magic number

File: relayer-cli/src/utils/proof.ts [hunk 3]
```diff
@@ -39,7 +39,7 @@ const getProofAtCount = async (chainid: number, nonce: number, count: number): P
   try {
     const subgraph = getSubgraph(chainid);
 
-    const result = await request(`https://api.studio.thegraph.com/query/67213/${subgraph}/version/latest`, query);
+    const result = await request(`https://api.studio.thegraph.com/query/85918/${subgraph}/version/latest`, query);
```
Comment (fcanela) [inline relayer-cli/src/utils/proof.ts:42]: Same suggestion about extracting to env vars

File: relayer-cli/src/utils/relay.ts [hunk 4]
```diff
@@ -25,7 +25,7 @@ const getCount = async (veaOutbox: VeaOutboxArbToEth, chainid: number): Promise<
   const stateRoot = await veaOutbox.stateRoot();
 
   const result = await request(
-    `https://api.studio.thegraph.com/query/67213/${subgraph}/version/latest`,
+    `https://api.studio.thegraph.com/query/85918/${subgraph}/version/latest`,
```
Comment (fcanela) [inline relayer-cli/src/utils/relay.ts:28]: Same suggestion about extracting to env vars

File: relayer-cli/src/utils/relay.ts [hunk 5]
```diff
@@ -119,7 +119,7 @@ const getNonceFrom = async (chainid: number, nonce: number, msgSender: string) =
     const subgraph = getSubgraph(chainid);
 
     const result = await request(
-      `https://api.studio.thegraph.com/query/67213/${subgraph}/version/latest`,
+      `https://api.studio.thegraph.com/query/85918/${subgraph}/version/latest`,
```
Comment (fcanela) [inline relayer-cli/src/utils/relay.ts:122]: Same suggestion about extracting to env vars

File: veascan-web/src/consts/bridges.ts [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import VeaInboxArbitrumSepolia from "@kleros/vea-contracts/deployments/arbitrumSepolia/VeaInboxArbToEthDevnet.json";
 import VeaOutboxSepolia from "@kleros/vea-contracts/deployments/sepolia/VeaOutboxArbToEthDevnet.json";
-import VeaInboxArbitrumSepoliaChiado from "@kleros/vea-contracts/deployments/arbitrumSepolia/VeaInboxArbToGnosisDevnet.json";
-import VeaOutboxArbitrumSepoliaChiado from "@kleros/vea-contracts/deployments/chiado/VeaOutboxArbToGnosisDevnet.json";
+// import VeaInboxArbitrumSepoliaChiado from "@kleros/vea-contracts/deployments/arbitrumSepolia/VeaInboxArbToGnosisDevnet.json";
```
Comment (fcanela) [inline veascan-web/src/consts/bridges.ts:3]: Is there any intentional reason to leave this commented code lines or did it just slip?

File: veascan-web/src/consts/bridges.ts [hunk 7]
```diff
@@ -42,22 +42,22 @@ export const bridges: IBridge[] = [
     to: sepolia.id,
     inboxAddress: VeaInboxArbitrumSepolia.address as `0x${string}`,
     inboxEndpoint:
-      "https://api.studio.thegraph.com/query/67213/veascan-inbox-arb-sep-devnet/version/latest",
+      "https://api.studio.thegraph.com/query/85918/veascan-inbox-arb-sep-devnet/v0.8.0",
```
Comment (fcanela) [inline veascan-web/src/consts/bridges.ts:45]: Same suggestion about extracting to env vars

File: veascan-web/src/consts/bridges.ts [hunk 8]
```diff
@@ -42,22 +42,22 @@ export const bridges: IBridge[] = [
...
-    to: gnosisChiado.id,
-    inboxAddress: VeaInboxArbitrumSepoliaChiado.address as `0x${string}`,
-    inboxEndpoint:
-      "https://api.studio.thegraph.com/query/67213/veascan-inbox-arb-sep-chiado-devnet/version/latest",
-    outboxAddress: VeaOutboxArbitrumSepoliaChiado.address as `0x${string}`,
-    outboxEndpoint:
-      "https://api.studio.thegraph.com/query/67213/veascan-outbox-arb-chiado-devnet/version/latest",
+      "https://api.studio.thegraph.com/query/85918/veascan-outbox-arb-sep-devnet/v0.8.0",
   },
+  // {
```
Comment (fcanela) [inline veascan-web/src/consts/bridges.ts:50]: Same comment regarding commented code

**Review by fcanela — APPROVED** [review]
I did an initial review without running code (I don't have the repo configured locally yet)     but it looks great. I've pointed minor maintenance improvements that can be made. Good job!

**Comment by fcanela** [comment]
I'm going to download the repo locally now and check everything in context. If I find anything, I'll update you.

File: veascan-web/src/consts/bridges.ts (hunk 6 above)
Comment (mani99brar) [reply veascan-web/src/consts/bridges.ts:3]: Commented to get veascan running for a single route for now, will uncomment with the updated deployments for the other route (Arbitrum Sepolia to Chiado).

Comment (fcanela) [reply veascan-web/src/consts/bridges.ts:3]: Thanks for the clarification, @mani99brar! I'll resolve the comment

File: relayer-cli/src/utils/proof.ts (hunk 2 above)
Comment (mani99brar) [reply relayer-cli/src/utils/proof.ts]: Done in [commit](https://github.com/kleros/vea/pull/305/commits/f01e4f920d5eb4179c069c41c583713e75d74709).

File: veascan-web/src/consts/bridges.ts (hunk 7 above)
Comment (mani99brar) [reply veascan-web/src/consts/bridges.ts:45]: Yep, will do.

File: .env.dist
Comment (jaybuidl) [inline .env.dist:1]: I wonder if this env file is meant be in the root folder and what code is supposed to use it? Presumably it's for the bots and veascan?

Comment (mani99brar) [reply .env.dist:1]: This .env file is specifically for veascan, the bots' environment variables are located in their respective folders. I tried to set up the .env in the veascan folder, but it isn't being loaded, likely due to how Parcel handles environment variables. If necessary, I can explore solutions to enable the use of the .env file directly within the veascan folder.

**Review by fcanela — DISMISSED** [review]
I reviewed it without performing any execution. Code wise it looks good.

File: .env.dist
Comment (jaybuidl) [reply .env.dist:1]: Fixed with https://github.com/kleros/vea/pull/305/commits/a9416968ed1823eccbaf919e8ca75bab86f4d151

**Review by jaybuidl — DISMISSED** [review]

File: veascan-web/scripts/runEnv.sh [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+    else
+        echo -e "${RED}✖${NC} $(basename $envFile)"
+    fi
+}
+
+envFile="$SCRIPT_DIR/../.env.${deployment}"
+sourceEnvFile "$envFile"
+sourceEnvFile "$envFile.public"
+
+(set -e; $commands)
```
[the remark this answers is not in the document]
Comment (jaybuidl) [reply veascan-web/scripts/runEnv.sh]: @coderabbitai check again

**Review by jaybuidl — APPROVED** [review]

**Pull request merged**